changelogs/fragments/302-ospf-static-neighbor.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - api_info, api_modify - add support for the ``routing ospf static-neighbor`` path in RouterOS 7 (https://github.com/ansible-collections/community.routeros/issues/302).
```

Comment:
```suggestion
  - api_info, api_modify - add support for the ``routing ospf static-neighbor`` path in RouterOS 7 (https://github.com/ansible-collections/community.routeros/pull/302).
```

Reply:
Thank you for noticing. Done, also in `300-add-ip-dhcp-server-matcher.yml` and `300-set-ip-dhcp-option-force-default.yml`.

Reply:
Thanks! You haven't changed it in this file though :)

Reply:
Whops, that wasn't intentional of course. Done.